.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/sig-rpc/sig-rpc.github.io/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Start a new [Discussion](https://github.com/sig-rpc/sig-rpc.github.io/discussions)
+- Provide as much context as you can, this will help reduce the need for us to ask for further information.
+- If relevant, include details such as your operating system or language/compiler version.
+
+We will then endeavour to respond to you as soon as possible.
+
+## I Want To Contribute
+
+> ### Legal Notice <!-- omit in toc -->
+> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the same licence as the existing content.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Reasonable Performance Computing SIG's Website & Knowledge-base, **including completely new features and minor improvements to existing functionality**. Following these guidelines will help maintainers and the community to understand your suggestion and find related suggestions.
+
+<!-- omit in toc -->
+#### Before Submitting an Enhancement
+
+- Make sure that you are using the latest version.
+- Read the [documentation](https://sig-rpc.github.io/) carefully and find out if the functionality is already covered, maybe by an individual configuration.
+- Perform a [search](https://github.com/sig-rpc/sig-rpc.github.io/issues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+- Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset. If you're just targeting a minority of users, consider writing an add-on/plugin library.
+
+<!-- omit in toc -->
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/sig-rpc/sig-rpc.github.io/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
```

Comment:
This is focused on software projects and doesn’t fully apply to our knowledge base. @Robadob, based on the suggestions from the RSECon workshop that you have looked through so far, do you have any points on what makes a helpful contribution that you could add here?